### PR TITLE
updating documentation

### PR DESCRIPTION
**Pull request by choldgraf**
I believe this addresses the addressable feedback in https://github.com/executablebooks/MyST-Parser/issues/146 , it updates a bunch of the documentation to try making it easier to follow and to find the right information.

### CODE REVIEW

**Comment by choldgraf** [comment]
cc @chrisjsewell in case he wants to take a look, but I'll plan to give this a merge in the next day or two unless somebody protests since this is a docs change

**Comment by chrisjsewell** [comment]
Looks good to me at first glance.
More of a sphinx-book-theme comment, but the logo+text takes up a lot of real-estate, IMO I'd rather it looked more like this:

image

**Comment by chrisjsewell** [comment]
Maybe remove some/all of the badges from the front page

File: docs/using/howto.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# How-to
```
Comment (chrisjsewell) [inline docs/using/howto.md:1]:
```suggestion
# How-To Guides
```

**Comment by choldgraf** [comment]
Thanks for the feedback, latest push updates the logo so it's more similar to the one you described, and also removes some of the more developer-focused badges from the index page 👍 I'm ready to merge if you are

**Pull request merged**